incubator/hnc/go.mod

```diff
@@ -9,19 +9,20 @@ require (
 	github.com/Azure/go-autorest/autorest/adal v0.6.0 // indirect
 	github.com/emicklei/go-restful v2.10.0+incompatible // indirect
 	github.com/go-logr/logr v0.1.0
+	github.com/go-openapi/spec v0.19.5 // indirect
```

Comment:
Any idea what caused these three new ones to be added?

Reply:
They are in the dependency chain so my guess is that it just hasn't been updated in a while.

```
go mod why -m go.uber.org/atomic github.com/go-openapi/spec golang.org/x/crypto
# go.uber.org/atomic
github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/cmd/manager
sigs.k8s.io/controller-runtime/pkg/log/zap
go.uber.org/zap
go.uber.org/atomic

# github.com/go-openapi/spec
github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/kubectl
k8s.io/cli-runtime/pkg/genericclioptions
k8s.io/cli-runtime/pkg/resource
k8s.io/cli-runtime/pkg/kustomize
sigs.k8s.io/kustomize/pkg/commands/build
sigs.k8s.io/kustomize/pkg/target
sigs.k8s.io/kustomize/pkg/transformers/config
github.com/go-openapi/spec

# golang.org/x/crypto
github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/kubectl
k8s.io/cli-runtime/pkg/genericclioptions
k8s.io/client-go/tools/clientcmd
golang.org/x/crypto/ssh/terminal
```